### DISCUSSION TITLE
Copilot invitation 404ing

### DISCUSSION THREAD

**Discussion by jsumners** [community/community#22432] [New to GitHub]
<p>When I click on the “View Invitation” button in the following email I get a <code>404</code>:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/2/6/26860c98d290be597c8df1ccfe1bc839c95c64f0.png" data-download-href="/uploads/short-url/5uNlgpkaiLWAm87z4wdZAYiAM0M.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/321201/181070858-e1e83abc-b410-407b-8489-5d1229bfd947.png" alt="image" data-base62-sha1="5uNlgpkaiLWAm87z4wdZAYiAM0M" width="491" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/2/6/26860c98d290be597c8df1ccfe1bc839c95c64f0_2_491x500.png, https://aws1.discourse-cdn.com/github/optimized/3X/2/6/26860c98d290be597c8df1ccfe1bc839c95c64f0_2_736x750.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/2/6/26860c98d290be597c8df1ccfe1bc839c95c64f0.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/2/6/26860c98d290be597c8df1ccfe1bc839c95c64f0_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">871×886 84.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>When I go back to the signup page I get a message that I’m already enrolled:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/0/b/0bf349dabcb69a34c8abb2713b2b73f5c45a09e5.png" data-download-href="/uploads/short-url/1HIuIclMMF7MhKHgx2YSgTcYGpL.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/321201/181070860-67e6dffe-3b33-4615-8a02-3dacc05a0f09.png" alt="image" data-base62-sha1="1HIuIclMMF7MhKHgx2YSgTcYGpL" width="690" height="432" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/0/b/0bf349dabcb69a34c8abb2713b2b73f5c45a09e5_2_690x432.png, https://aws1.discourse-cdn.com/github/original/3X/0/b/0bf349dabcb69a34c8abb2713b2b73f5c45a09e5.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/0/b/0bf349dabcb69a34c8abb2713b2b73f5c45a09e5.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/0/b/0bf349dabcb69a34c8abb2713b2b73f5c45a09e5_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">977×612 67.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>But when I try to view the community repo I get another <code>404</code>:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/4/d/4db554f8f990e9968cd4e3dea461d331184001dd.jpeg" data-download-href="/uploads/short-url/b5rgyEFuN8pxbfTWluHFxGgjbEp.jpeg?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/321201/181070861-070f27d0-900c-408e-9107-4d35e230d4f2.jpeg" alt="image" data-base62-sha1="b5rgyEFuN8pxbfTWluHFxGgjbEp" width="690" height="431" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/4/d/4db554f8f990e9968cd4e3dea461d331184001dd_2_690x431.jpeg, https://aws1.discourse-cdn.com/github/original/3X/4/d/4db554f8f990e9968cd4e3dea461d331184001dd.jpeg 1.5x, https://aws1.discourse-cdn.com/github/original/3X/4/d/4db554f8f990e9968cd4e3dea461d331184001dd.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/4/d/4db554f8f990e9968cd4e3dea461d331184001dd_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">848×530 72 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Am I in this preview or not?</p>

**Comment by fbidu** [comment]
<p>yeah, same here! Just saw the e-mail but all links are 404ing on me</p>

**Comment by jsumners** [comment, marked as answer]
<p>Got a new invitation today that has solved the issue for me.</p>

**Comment by liudonghua123** [comment]
<p>I have the same issue, is the new invitation auto resend?</p>

**Comment by ppamorim** [comment]
<p>I am experiencing the same issue and I am not receiving the invitation again.</p>

**Comment by artonio** [comment]
<p>Same here. All links are 404 for me</p>
